Source/WebCore/accessibility/isolatedtree/AXIsolatedObject.cpp

```diff
@@ -122,6 +122,8 @@ void AXIsolatedObject::initializeProperties(const Ref<AccessibilityObject>& axOb
             setProperty(AXProperty::ListMarkerText, object.listMarkerText().isolatedCopy());
             setProperty(AXProperty::ListMarkerLineID, object.listMarkerLineID());
         }
+
+        setProperty(AXProperty::Language, object.language().isolatedCopy());
```

Comment:
does this need to be gated by text thread apis?

If language() isEmpty() do we want to avoid doing an isolated copy and store?

Reply:
> does this need to be gated by text thread apis?

Ah, no it doesn't, good catch!

> If language() isEmpty() do we want to avoid doing an isolated copy and store?

Fixed!